packages/create-vite/index.js

```diff
@@ -254,14 +254,17 @@ async function init() {
 
   write('package.json', JSON.stringify(pkg, null, 2))
 
-  const pkgManager = /yarn/.test(process.env.npm_execpath) ? 'yarn' : 'npm'
+  const pkgInfo = pkgFromUserAgent(process.env.npm_config_user_agent)
+  const pkgManager = pkgInfo ? pkgInfo.name : 'npm'
```

Comment:
```suggestion
  const pkgManager = pkgInfo?.name ?? 'npm'
```

Reply:
`?.` & `??` maybe not support?
I wrote it this way before, and some checks failed.

Reply:
Yes, this file is not transpiled